client/dfclient/src/util.cpp

```diff
@@ -16,7 +16,7 @@ void SendData(flatbuffers::FlatBufferBuilder& builder) {
 	auto size = builder.GetSize();
 	auto str = std::string(data, data + size);
 
-	gameData.socket->Send(str);
+	webSocketManager._ws->Send(str);
```

Comment:
Functionality: https://github.com/capturcus/deadfish/blob/a56b3db6c6e3560ff7e9bb9ad0534208d1207837/client/dfclient/src/websocket.cpp#L8
Use: https://github.com/capturcus/deadfish/blob/a56b3db6c6e3560ff7e9bb9ad0534208d1207837/client/dfclient/src/state_manager.cpp#L65

Reply:
OK, `GetMessages` is useful, but it can just be a function which takes `_ws` as an additional argument. I don't see why this has to be a struct. It's ugly if "guts" of such manager are open for everybody to use.

Reply:
Got rid of the WebSocketManager. I think that originally there was some purpose to it but it was lost after a few refactors.